core/js/src/files/PathsJs.kt

```diff
@@ -92,6 +92,20 @@ internal class FileSource(private val path: Path) : RawSource {
     private var buffer: dynamic = null
```

Comment:
Just curious, why aren't you using the `Buffer` type from kotlin-wrappers for the buffer instances?

Reply:
It's a JS buffer returned by the [fs.readSync](https://nodejs.org/api/fs.html#fsreadsyncfd-buffer-options) and it's incompatible with `kotlinx.io.Buffer`.

The way it all works could be improved (by removing that buffer altogether and using an array from a buffer passed to the `readAtMostTo` call as a destination buffer supplied to `fs.readSync`).